Units/parser-iniconf.r/simple-pythonLoggingConfig.d/expected.tags

```diff
@@ -1,3 +1,6 @@
+"key in double quotes"	input.conf	/^"key in double quotes" = 1$/;"	key	language:Iniconf	section:[toml_array_of_tables]
+'key in single quotes'	input.conf	/^'key in single quotes' = 2$/;"	key	language:Iniconf	section:[toml_array_of_tables]
+[toml_array_of_tables]	input.conf	/^[[toml_array_of_tables]]$/;"	section	language:Iniconf
```

Comment:
This is about the hack.
Instead of extending an existing test case, add a new one.
e.g. adding:

 Units/parser-iniconf.r/toml-support-in-geany.d/{args.ctags,input.toml,exected.tags}

Putting `--langmap=Iniconf:.toml` to args.ctags can make ctags use the Initconf to parser input.toml.

Reply:
Done.